dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/programstageworkinglist/JsonProgramStageQueryCriteria.java

```diff
@@ -49,7 +49,7 @@ default JsonDatePeriod getEventOccurredAt() {
     return get("eventOccurredAt").as(JsonDatePeriod.class);
   }
 
-  default boolean getFollowUp() {
+  default Boolean getFollowUp() {
     return getBoolean("followUp").booleanValue();
```

Comment:
This booleanValue() function returns a primitive boolean though. Not sure how this default function gets consumed in the end though.

Reply:
Firstly, this will only be used in tests. Secondly, there are three possible cases:

`followUp = null` (in this case, it will be ignored by JSON).
`followUp` is not provided (it will also be ignored by JSON).
`followUp = true/false` (the provided value will be used as-is).